guides/whitelist.htm

```diff
@@ -22,7 +22,6 @@ <h5>Please note:</h5>
                     <p>Any BOINC work done outside of the whitelisted BOINC projects will not be rewarded Gridcoin.</p>
                     <p>Want a project whitelisted? Ask about it on the <code>#whitelisting-committee</code> channel on the Gridcoin Discord! More details about adding a project to the whitelist can be found on the <a href="/wiki/whitelist-process.html">whitelist process wiki page</a></p>
                     <p>You can check whitelisted project details on <a href="https://gridcoinstats.eu/project">Gridcoinstats</a>.</p>
-                    <p>The whitelist is verifiable on the wallet on the projects tab on the beacon menu or with the <code>listprojects</code> command</p>
```

Comment:
why is this being removed? It is accurate

Reply:
Because I moved it down further on the page